### DISCUSSION TITLE
What are the units of TextBlock.LetterSpacing?

### DISCUSSION THREAD

**Discussion by kuiperzone** [AvaloniaUI/Avalonia#17593] [Q&A]
1/1000 em? Or pixels?

And I note that there is no reference to it in: https://reference.avaloniaui.net/api/Avalonia.Controls/TextBlock/

**Comment by kuiperzone** [comment]
It appears to be pixels. Am I correct?

**Reply by timunie** [reply]
Best to lookup in source. Reference page is outdated sadly.
